test/sanity/code-smell/use-argspec-type-path.py

```diff
@@ -11,6 +11,7 @@ def main():
         'lib/ansible/modules/cloud/lxc/lxc_container.py',
         'lib/ansible/modules/cloud/rackspace/rax_files_objects.py',
         'lib/ansible/modules/database/mongodb/mongodb_parameter.py',
+        'lib/ansible/modules/database/mongodb/mongodb_role.py',
```

Comment:
Instead of ignoring linter errors, do as error message suggests:
```
lib/ansible/modules/database/mongodb/mongodb_role.py:194:47: use argspec type="path" instead of type="str" to avoid use of `expanduser`
```

you can put the path to config into module args with `path` type and default, which you have hardcoded.